freva-data-portal-worker/src/data_portal_worker/load_data.py

```diff
@@ -185,16 +176,24 @@ def from_object_path(
         ).dict()
         expires_in = str_to_int(os.environ.get("API_CACHE_EXP"), 3600)
         status_dict["status"] = 3
+        dset: Optional[xr.Dataset] = None
         self.cache.setex(path_id, expires_in, cloudpickle.dumps(status_dict))
```

Comment:
changing the name here as `self.cache.setex(f"{path_id}-metadata", expires_in, cloudpickle.dumps(status_dict))` could be an idea to not get confused in bug fixing.

Reply:
Unfortunately, that's not as simple. Because this also needs to be handled on the rest-api side. I'd like to keep this as this this for the time being.